cellxpredict/visualization.py

```diff
@@ -1,37 +1,39 @@
+from typing import List
+
 import numpy as np
 from skimage.filters import median as median_filter
 from skimage.morphology import disk
 
-from typing import List
 
 def float_to_uint8(input: np.ndarray[np.float32]) -> np.ndarray[np.uint8]:
-    '''
+    """
     Transforms an np.float32 image into a np.uint8 image
```

Comment:
```suggestion
    """Transforms an np.float32 image into a np.uint8 image
```

No newline after the triple quotes